pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py

```diff
@@ -92,13 +92,23 @@ def setUpModule():  # pylint:disable=invalid-name
     .. _Pulp #1759: https://pulp.plan.io/issues/1759
     """
     set_up_module()
-    if selectors.bug_is_untestable(1759, config.get_config().version):
+    cfg = config.get_config()
+    if selectors.bug_is_untestable(1759, cfg.version):
         raise unittest.SkipTest('https://pulp.plan.io/issues/1759')
+    cmd = 'semanage boolean --modify --on pulp_manage_rsync'.split()
+    if not utils.is_root(cfg):
+        cmd.insert(0, 'sudo')
+    cli.Client(cfg).run(cmd)
 
 
 def tearDownModule():  # pylint:disable=invalid-name
     """Delete orphan content units."""
-    api.Client(config.get_config()).delete(ORPHANS_PATH)
+    cfg = config.get_config()
+    api.Client(cfg).delete(ORPHANS_PATH)
+    cmd = 'semanage boolean --modify --off pulp_manage_rsync'.split()
```

Comment:
Would not worth getting this logic as a helper function? The helper function can receive a boolean indicating to turn on and off and the config object.

This is not a blocker.